inst/@sym/private/do_highbyte_escapes.m

```diff
@@ -48,14 +48,7 @@
   end
 
   % get the two-char hex numbers make them into bytes
-  if (exist ('OCTAVE_VERSION', 'builtin') && ...
-      compare_versions (OCTAVE_VERSION (), '4.3.0', '<'))
-    % Bug on old Octave: https://savannah.gnu.org/bugs/?49659
-    dec = char(hex2dec(NM.hex));
-  else
-    % roughly 3-4 times slower than the above
-    dec = char (hex2dec (struct2cell (NM)));
-  end
+  dec = char (hex2dec (struct2cell (NM)));
```

Comment:
According to https://savannah.gnu.org/bugs/?49659, the bug was fixed in version 4.4 (2018-04-30). One can check that the fix is in version 5.1 https://hg.savannah.gnu.org/hgweb/octave/file/d05d6eebde10/libinterp/corefcn/regexp.cc as well. LGTM!